File: 1672. Richest Customer Wealth/solution.cs
public class Solution
{
    public int MaximumWealth(int[][] accounts)
    {
        int maxWealth = 0;

        foreach (var account in accounts)
        {
            int wealth = 0;

            for (int i = 0; i < account.lenght; i++)
            {
                wealth += account[i];
            }

            if(wealth > maxWealth)
            {
                maxWealth = wealth;
            }

        }

        return maxWealth; 

    }

}

File: 1672. Richest Customer Wealth/solution.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function (accounts) {
  let maxWealth = 0;
  accounts.forEach((account) => {
    let wealth = 0;
    for (let i = 0; i < account.length; i++) {
      wealth += account[i];
    }
    wealth > maxWealth ? (maxWealth = wealth) : maxWealth;
  });

  return maxWealth;
};
